rllib/models/catalog.py

```diff
@@ -241,8 +241,8 @@ def get_action_dist(
             if action_space.dtype.name.startswith("int"):
                 low_ = np.min(action_space.low)
                 high_ = np.max(action_space.high)
-                assert np.all(action_space.low == low_)
-                assert np.all(action_space.high == high_)
+                assert np.all(action_space.low >= low_)
```

Comment:
Hi @frenkowski and @richardliaw,

I am looking at these assertions and wondering how they would be false? I cannot think of how but I am probably missing something. Do you have an example?

Reply:
Sure!
If you have a Box with independent bound for each dimension (e.g. `Box(low=np.array([-1.0, -2.0]), high=np.array([2.0, 4.0]), dtype=np.float32)`), the lowest value is `-2.0`. 
Therefore in this case the assertion fails, given that `-1.0 != -2.0` (but `-1.0 >= -2.0`).
The same obviously applies to the highest value.

Reply:
Hi @frenkowski,

Thank you for the reply. I think I may not have been clear. I agree that the original issue you posted is a bug what I am wondering is after your fix would it ever assert. 

You defined low_ to be np.min(action_space.low) then you have an assertion to check that all values in action_space.low are greater than our equal to that value. By construction that is the lowest value so how could that ever be false?  

I am not saying the logic of your solution is wrong I am just wondering what kind of issue that new assertion could catch.

Reply:
@mvindiola1 I apologize for the misunderstanding!
I totally agree with you, we can just remove the assertions regardless of the bug.
I suggested that change because I sincerely do not know the logic of the author with respect to the original assertion.

Reply:
I guess it is up to @richardliaw as to which approach he prefers.

Reply:
hmm, agree with @mvindiola1 that we can just remove the assertions!

Reply:
@mvindiola1 @richardliaw I removed the assertions with the new [commit](https://github.com/ray-project/ray/pull/16860/commits/bc8bc924a9bc5e4e99f45674c06fdf16e6913942)